client/layout/guided-tours/tours/media-basics-tour/index.js

```diff
@@ -61,7 +61,7 @@ export const MediaBasicsTour = makeTour(
 				<Fragment>
 					<p>
 						{ translate(
-							'You can also drag-and-drop image and video files from your computer into your media library.'
+							'You can also drag-and-drop image and video files from your computer into your Media Library.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.